python/modules/IcePy/Util.cpp

```diff
@@ -271,8 +271,8 @@ IcePy::PyException::PyException() { ex = PyErr_GetRaisedException(); }
 
 IcePy::PyException::PyException(PyObject* raisedException)
 {
-    Py_XINCREF(raisedException);
```

Comment:
I added the assert in case `Py_XINCREF` was used on purpose, for example we pass nullptr. By looking at the code seems we do not.